docs/deployment/kubernetes.md

```diff
@@ -20,21 +21,22 @@ We have a special guide for minikube here:
 
 * [Getting started with OpenFaaS on minikube](https://medium.com/devopslinks/getting-started-with-openfaas-on-minikube-634502c7acdf)
 
-Are you using Google Kubernetes Engine (GKE)? You'll need to create an RBAC role with the following command:
+!!! tip
+    Are you using Google Kubernetes Engine (GKE)? You'll need to create an RBAC role with the following command:
 
-```
-# kubectl create clusterrolebinding "cluster-admin-$(whoami)" \
-   --clusterrole=cluster-admin \
-   --user="$(gcloud config get-value core/account)"
-```
+    ```bash
```

Comment:
Curious why this is indented and block-quoted at the same time?

Reply:
You have to do this so the code block will get put into the Tip box as a code block.
This is what it looks like rendered:
![screen shot 2018-03-29 at 5 19 34 pm](https://user-images.githubusercontent.com/1606945/38116380-68eed87e-3375-11e8-8761-207a6782a149.png)

Reply:
You have to do this so the code block will get put into the Tip box as a code block.
This is what it looks like rendered:
![screen shot 2018-03-29 at 5 19 34 pm](https://user-images.githubusercontent.com/1606945/38116380-68eed87e-3375-11e8-8761-207a6782a149.png)